Content.Server/Physics/Controllers/ConveyorController.cs

```diff
@@ -152,8 +152,13 @@ private void AwakenEntities(ConveyorComponent component)
                 {
                     if (!bodyQuery.TryGetComponent(entity, out var physics))
                         continue;
+                    if (!xformQuery.TryGetComponent(entity, out var intersectingXform))
+                        continue;
 
-                    _physics.WakeBody(physics);
+                    if (!intersectingXform.Anchored)
+                    {
+                        _physics.WakeBody(physics);
+                    }
```

Comment:
Are you sure this works as intended, because anchored bodies shouldn't be able to wake anyway

Reply:
![BuQ6QRgUGb](https://user-images.githubusercontent.com/11891037/194288812-39731687-9d60-4b3d-81fc-64a1be52628c.gif)
I am sure, the issue is that WakeBody implementation firstly calls `SetCanCollide(body, true)`. The only check I see that can make the set fail is a `IsEntityOrParentInContainer` check, which is completely unrelated. I guess the root issue is that WakeBody does two things: it enables collisions and actually wakes the body. The bug is related to the first part of WakeBody which enables collisions.

Reply:
If there's any other test you can suggest me to do to verify nothing else could be breaking, I can do it

Reply:
I think there's two blast doors there yeah? It's weird something is effecting the outer one and not the inner one.

Reply:
> I think there's two blast doors there yeah? It's weird something is effecting the outer one and not the inner one.

uh it isn't? I just stopped the conveyor belt since one blast door being passed by the crate was enough to prove the fix works haha, somehow i stopped the belt in the perfect moment

Reply:
> > I think there's two blast doors there yeah? It's weird something is effecting the outer one and not the inner one.
> 
> uh it isn't? I just stopped the conveyor belt since one blast door being passed by the crate was enough to prove the fix works haha, somehow i stopped the belt in the perfect moment

Ohhhh oops